packages/core/src/utils/styleUtils/styleTransformers.ts

```diff
@@ -16,6 +16,7 @@ export const transformVisibility = (value?: boolean): CSSProperties => {
   return {};
 };
 
+// TODO: Remove in next major version v2 since the change is 17 months old
```

Comment:
I think the change is 12 months old, not 17.

<img width="680" alt="Screenshot 2025-04-25 at 10 28 57" src="https://github.com/user-attachments/assets/756b7e28-9d8e-433c-9c0d-43d83ccf5e9a" />

This is a tricky scenario, it would only impact any component dropped before Apr 2024 that is still using the default value for those styles :/ Probably a low probability of that being the case, but we should explain that in the release notes of the major version.

Reply:
Good call. I will mention that in the release notes.